src/Umbraco.Web/Templates/TemplateUtilities.cs

```diff
@@ -15,16 +14,11 @@
 
 namespace Umbraco.Web.Templates
 {
-    //NOTE: I realize there is only one class in this namespace but I'm pretty positive that there will be more classes in
-    //this namespace once we start migrating and cleaning up more code.
 
-    /// <summary>
-    /// Utility class used for templates
-    /// </summary>
+    [Obsolete("This class is obsolete, all methods have been moved to other classes such as InternalLinkHelper, UrlResolver and MediaParser")]
```

Comment:
MediaParser is named ImageSourceParser.. 

There also is a lot of variables that in named inconsistent now.. Think we need to clean this up, before we continue.